addons/website/models/ir_qweb.py

```diff
@@ -101,10 +101,9 @@ def _prepare_frontend_environment(self, values):
         # update options
 
         irQweb = irQweb.with_context(website_id=current_website.id)
-
         if 'inherit_branding' not in irQweb.env.context and not self.env.context.get('rendering_bundle'):
             if editable:
-                # in edit mode add brancding on ir.ui.view tag nodes
+                # in edit mode add branding on ir.ui.view tag nodes
```

Comment:
I would have avoided the 2 "typo" changes in this file on which you don't change anything other than that but let's move forward :+1: 
Just let's keep that for master next time (it's ok in file you touch)

Reply:
@rdeodoo Added a test for this in <https://github.com/odoo/odoo/pull/156479>.